win64.mak

```diff
@@ -520,8 +493,8 @@ clean:
 	del $(LIB)
 	del phobos.json
 
-install: phobos.zip
-	$(CP) phobos.lib phobos64.lib $(DIR)\windows\lib
+install: phobos.zip $(LIB)
+	copy /y /b $(LIB) $(DIR)\windows\lib
```

Comment:
Looks like this is unused - it a) didn't make sure the lib is actually built beforehand, and b) tried to copy a `phobos.lib`, which should have never existed (no `MODEL` suffix).